### PR TITLE
Reorder people on team page using hour sheet

### LINKED ISSUE #226: Reshuffle Team page by hours
Based on who's contributed the most (by hours). We should take out people who haven't contributed.

### PR DESCRIPTION
**Pull request by AndrewLester**
Resolve #226

### CODE REVIEW

**Comment by ErikBoesen** [comment]
Conflicts

File: _pages/team.html [hunk 1]
```diff
@@ -69,73 +79,63 @@ <h5>Stephanie Synnott</h5>
...
-            <p>Jack is a mechanic.</p>
+          <img src="/assets/img/team/fernanda-molina.png">
+          <h5>Fernanda Molina</h5>
+          <p>Fernanda is a programmer with experience in Java and Python. She works on helping fix issues with the robot code and updating the team's website.</p>
         </div>
         <div>
-            <img src="/assets/img/team/nikita-solovyov.png">
-            <h5>Nikita Solovyov</h5>
-            <p>Nikita works on scouting analysis.</p>
+            <img src="/assets/img/team/jakub-leski.png">
```
Comment (ErikBoesen) [inline _pages/team.html]: Honestly, remove everyone with <100 hours if you like. Do it in a separate commit though.

Comment (AndrewLester) [reply _pages/team.html]: On the same pull?

Comment (AndrewLester) [reply _pages/team.html]: Oof. Maybe put the cutoff at 75?

Comment (ErikBoesen) [reply _pages/team.html]: Sure, 75 is okay.

Comment (ErikBoesen) [reply _pages/team.html]: On second thought, let's not make cuts on the website until we know more about who will be on next year's team.
